### ISSUE TITLE
[Accessibility][Auto-suggest] Auto-suggest default input is missing aria-expanded attribute

### ISSUE THREAD

**Issue by hchokshi** [microsoft/fast-dna#3009]
**Describe the bug; what happened?**
https://github.com/microsoft/fast-dna/blob/0302bb42eba37a1e032692fa14f3edcd2772ccde/packages/react/fast-components-react-base/src/auto-suggest/auto-suggest.tsx#L279

The auto-suggest input renders as a `combobox` aria-role, which should have an expanded attribute. Likely should be `aria-expanded={state.isMenuOpen}`.

E.g. select, which also behaves as a combobox, has the same expanded property added here
https://github.com/microsoft/fast-dna/pull/2393/files#diff-8e762c110a4e910d7c66c5c6bbb22bd4L171

**What are the steps to reproduce the issue?**
Render an AutoSuggest component without setting the input region prop.

**What behavior did you expect?**
`aria-expanded` attribute indicating menu open/close state.

![image](https://user-images.githubusercontent.com/10136407/80254422-f6c65780-862f-11ea-9727-81beb4820b29.png)

**Comment by chrisdholt** [comment]
Thanks @hchokshi this does appear to be a bug!
